shakenfist/daemons/cluster.py

```diff
@@ -61,7 +52,7 @@ def _cluster_wide_cleanup(self, last_loop_run):
         LOG.info('Running cluster maintenance')
 
         # Cleanup soft deleted objects
-        for objtype in OBJECT_NAMES:
+        for objtype in OBJECT_NAMES_TO_CLASSES:
             for _, objdata in etcd.get_all(objtype, None):
                 try:
                     obj = OBJECT_NAMES_TO_CLASSES[objtype].from_db(
```

Comment:
Setting `OBJECT_NAMES_TO_CLASSES` to contain actual classes causes circular references and leads to multiple code locations holding the same data.

Maybe change this to utilise `globals()`?
```
obj = globals()[objtype].from_db(
```
The classes in `cluster.py` will have be to imported directly 
```
from shakenfist.instance import Instance
```

Reply:
I don't want the class here though, just its name. I feel like I could do something terrible with importlib in baseobjectmapping.py, but I wasn't particularly happy with that option and bravely ran away.

Reply:
You do need the class here, in order to call `.from_db()`